backend/lib/kubernetes-client/index.js

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright (c) 2019 by SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+'use strict'
+
+const util = require('./util')
+const Client = require('./Client')
+const kubeconfig = require('../kubernetes-config')
+const { isHttpError } = util
+const config = kubeconfig.load(process.env)
+
+function createClient (options) {
```

Comment:
make private, should not be called externally with privileged: true

Reply:
✔️